quickevent/app/plugins/qml/Event/src/Event/eventconfig.cpp

```diff
@@ -106,13 +106,13 @@ void EventConfig::save(const QString &path_to_save)
 			val_str = val.toString();
 		q_up.bindValue(":key", key);
 		q_up.bindValue(":val", val);
-		q_up.exec();
+		q_up.exec(qf::core::Exception::Throw);
```

Comment:
Maybe try-catch around all the sql calls will be better. I agree that sql calls should be handled consistently